sympy/printing/tests/test_pycode.py

```diff
@@ -2,15 +2,18 @@
 from __future__ import (absolute_import, division, print_function)
 
 from sympy.codegen import Assignment
-from sympy.core import Expr, Mod, symbols, Eq, Le, Gt
+from sympy.core import Expr, Mod, symbols, Eq, Le, Gt, zoo, oo
 from sympy.core.numbers import pi
+from sympy.external import import_module
```

Comment:
`import_module` & `skip` are no longer neccessary (we're not using numpy here)

Reply:
Oh, Sorry..!! Forgot to remove it..
Has been corrected though..
@bjodah